src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -346,6 +364,17 @@ bool PerformInteractionRequest::ProcessVRResponse(
     return false;
   }
 
+  if (first_responser_ == FirstPerformInteractionResponser::VR) {
+    // After VR.PerformInteraction response HMI should close UI popup window
+    // if UI.PerformInteraction response comes
+    // after VR.PerformInteraction response.
+    // In this case SDL should send UI_ClosePopUp request.
+    smart_objects::SmartObject hmi_request_params =
+        smart_objects::SmartObject(smart_objects::SmartType_Map);
+    hmi_request_params[hmi_request::method_name] = "UI.PerformInteraction";
+    SendHMIRequest(hmi_apis::FunctionID::UI_ClosePopUp, &hmi_request_params);
```

Comment:
@YarikMamykin Do we need to send it independently from response (success or failure)?

Reply:
@YarikMamykin In this case update PR description accordingly